pages/docs/engine/ra-optimisation/branch-and-bound-optimisation/branch-and-bound/linear-rao.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+title: Linear RAO
+permalink: /docs/engine/ra-optimisation/branch-and-bound/linear-rao
+hide: true
+feature-img: "assets/img/Hans_Otto_Theater_Potsdam_-_fake_colors_cut.jpg"
+tags: [Docs, Branch & Bound RAO]
+---
+
+### Overview
+
+The Linear RAO optimizes the Range Actions (for now, only PSTs) according to a configurable objective function (e.g. maximization of the minimum margin). The solving algorithm approximates the impact of the Remedial Actions on the network flows with linear sensitivity coefficients. It therefore solves [linear optimisation problems](/docs/engine/ra-optimisation/branch-and-bound/linear-optimisation-problem) to find the optimal application of the Remedial Actions. Moreover, it can iterate over several reference points in order to mitigate the linear approximation inherent to its optimisation problems.
```

Comment:
linear optimisation problem(s) in the link

Reply:
Do you mean that I should put the link to the linear optimisation problem page here too?

Reply:
No sorry I just meant "remove the "s" " :laughing: